qcodes/instrument_drivers/Keysight/keysight_e4980a.py

```diff
@@ -0,0 +1,333 @@
+            get_parser=float,
+            unit="V",
+            vals=Numbers(0, 20),
+            docstring="Gets and sets the voltage level for measurement signal."
+        )
+
+        self.add_parameter(
+            "impedance",
+            get_cmd=self._get_complex_impedance,
+        )
+
+        self.add_parameter(
+            "measurement_function",
+            get_cmd=":FUNCtion:IMPedance?",
+            set_cmd=self._set_measurement
+        )
+
+        self.add_parameter(
+            "measure",
+            get_cmd=self._measurement,
+        )
+
+        self.add_parameter(
+            "range",
+            get_cmd=":FUNCtion:IMPedance:RANGe?",
+            set_cmd=":FUNCtion:IMPedance:RANGe {}",
+            unit='Ohm',
+            vals=Enum(0.1, 1, 10, 100, 300, 1000, 3000, 10000, 30000, 100000),
+            docstring="Selects the impedance measurement range, also turns "
+                      "the auto range function OFF."
+        )
+
+        self.add_parameter(
+            "system_errors",
+            get_cmd=":SYSTem:ERRor?",
+            docstring="Returns the oldest unread error message from the event "
+                      "log and removes it from the log."
+        )
+
+        self.add_submodule(
+            "_correction",
+            Correction4980A(self, "correction")
+        )
+
+        self.connect_message()
+
+    @property
+    def correction(self):
+        return self.submodules['_correction']
+
+    def _get_complex_impedance(self) -> MeasurementPair:
+        """
+        Returns the impedance in the format of (R, X), where R is the
+        resistance, and X is the reactance.
+        """
+        measurement = self.ask(":FETCH:IMPedance:CORRected?")
+        r, x = [float(n) for n in measurement.split(",")]
+        measurement_pair = MeasurementPair(E4980AMeasurements.RX)
+        measurement_pair.set((r, x))
+        return measurement_pair
+
+    def _measurement(self) -> MeasurementPair:
+        """
+        Returns a measurement result with the selected measurement function.
+        """
+        measurement = self.ask(":FETCH:IMPedance:FORMatted?")
+        val1, val2, _ = [float(n) for n in measurement.split(",")]
+        measurement_pair = MeasurementPair(self._measurement_function)
+        measurement_pair.set((val1, val2))
+        return measurement_pair
+
+    def _set_measurement(self,
+                         measurement_function: MeasurementFunction) -> None:
+        """
+        Selects the measurement function.
+        """
+        self._measurement_function = measurement_function
+        self.write(f":FUNCtion:IMPedance {measurement_function.name}")
+
+    def clear_status(self) -> None:
+        """
+        Clears the following:
+            • Error Queue
+            • Status Byte Register
+            • Standard Event Status Register
+            • Operation Status Event Register
+            • Questionable Status Event Register (No Query)
```

Comment:
let's avoid unicode symbols? so that sphinx can render this as a bullet point list in the docs

Reply:
Good point. I have removed the "dots", and I "think" I fixed all the other sphinx error.